### DISCUSSION TITLE
Livewire @entangle issue with mutliple post

### DISCUSSION THREAD

**Discussion by stanislawszenk** [livewire/livewire#4684] [1. Help] [answered]
<br />
I'm actually using a component I made using Livewire and AlpineJS.<br />
The issue I got is the following :<br />
I have a button to add a new section, in this section I can wrote title and description (wysiwyg editor) so to handle this I have in my component :

    <div
        x-data="{content: @entangle($attributes->wire('model')),...setupEditor()}"
        x-init="() => init($refs.editor)"
        wire:ignore
        {{ $attributes->whereDoesntStartWith('wire:model') }}>

And in my livewire php file :

    protected $rules = [
        'newLesson.*.title' => 'required|string',
        'newLesson.*.lesson_content' => 'required|string',
        'newLesson.*.type_content' => 'required|string',
    ];

So in my livewire blade I have something like this :

    <x-editor wire:model.lazy="newLesson.{{$i}}.lesson_content"></x-editor>

But when I try, I've got an error in my console :

> SupportAlpine.js:102 Livewire Entangle Error: Livewire property
> 'newLesson.4.lesson_content' cannot be found

Did some of you already got this kind of issue ?
I don't really know how to handle it actually.
Thanks for your time and help :)

**Comment by joshhanley** [comment, marked as answer]
Worked through the issue on discord and solved it 🙂

For anyone else coming across it, the `addLesson` method was referencing `$newLessons` instead of `$newLesson`.
